### ISSUE TITLE
Old Kustomize Version

### ISSUE THREAD

**Issue by jwenz723** [fluxcd/flux#2347]
**Describe the bug**

It appears that the kustomize version that is being packaged into the fluxcd/flux docker image does not correlate to the version specified in [kustomize.version](https://github.com/fluxcd/flux/blob/1728c8d64705a7e809c90a2c864d492fb131b43a/docker/kustomize.version#L1)

I would expect the version to be `3.1.0` but it is actually `2.0.3`.

**To Reproduce**
Steps to reproduce the behaviour:

Bare docker:

```
$ docker run --rm -it --entrypoint "" fluxcd/flux:1.13.3 kustomize version
Version: {KustomizeVersion:2.0.3 GitCommit:a6f65144121d1955266b0cd836ce954c04122dc8 BuildDate:2019-03-05T20:37:42Z GoOs:linux GoArch:amd64}
```

Running in k8s:

```
# install fluxd into k8s using fluxcd/flux:1.13.3 docker image then the following
$ kubectl exec -it flux-54b67d64f6-nwj5b -- fluxd --version
1.13.3
$  kubectl exec -it flux-54b67d64f6-nwj5b -- kustomize version
Version: {KustomizeVersion:2.0.3 GitCommit:a6f65144121d1955266b0cd836ce954c04122dc8 BuildDate:2019-03-05T20:37:42Z GoOs:linux GoArch:amd64}
```

**Expected behavior**

I would expect the kustomize version to be 3.1.0

**Logs**
If applicable, please provide logs of `fluxd` or the helm-operator. In a standard stand-alone installation of Flux, you'd get this by running `kubectl logs -n default deploy/flux`.

n/a

**Additional context**
Add any other context about the problem here, e.g 
- Flux version: 1.13.3
- Helm Operator version: n/a
- Kubernetes version: n/a
- Git provider: n/a
- Container registry provider: n/a

**Comment by jwenz723** [comment]
It appears that the latest flux-prerelease docker image has the v3.1.0 of kustomize installed:

```
$ docker run --rm -it --entrypoint "" fluxcd/flux-prerelease:master-1728c8d6-wip kustomize version
Version: {KustomizeVersion:3.1.0 GitCommit:95f3303493fdea243ae83b767978092396169baf BuildDate:2019-07-26T18:11:16Z GoOs:linux GoArch:amd64}
```

Any estimate on when a stable non-prerelease version of the flux image will be release with v3.1.0 kustomize?

**Comment by stefanprodan** [comment]
The Kustomize and kubectl updates are scheduled for 1.14 release https://github.com/fluxcd/flux/milestones

We haven't set a fix date for 1.14 but you can see the releases cadence here https://github.com/fluxcd/flux/releases
